test/table.js

```diff
@@ -0,0 +1,45 @@
+import {smartTable} from 'smart-table-core';
+import stMixins from '../dist/smart-table-vue.js';
+const {table: tableMixin} = stMixins;
+
+export default (test) => {
+    const shallowMount = window.VueTestUtils.shallowMount;
```

Comment:
I don't know how to import this with ES6 import and rollup without side-effects (ie. Vue, VueTemplateCompiler, nodejs builtins or globals).

Reply:
Maybe you can treat your test as if it was a regular app and bundle it within all its dependencies. So that in your output, you have only one file. 

[here is an example](https://github.com/lorenzofox3/lls-test/tree/master/app/test). In that case it uses React (and it is quite inefficient) but I believe you can achieve the same thing with Vue.js: 

1. one entry point for your testing program (``test/index.js``) which you bundle with all its dependencies (in ``test/dist/debug.js``)
2. While developing, you can use ``test/dist/index.html`` with your favorite browser
3. For CI, you might need to start a file server and ask puppeteer to visit the testing page ``test/dist/index.html``

That is an idea, don't hesitate to innovate or to go by another path based on how some other project does